package.json

```diff
@@ -13,7 +13,8 @@
     "start": "webpack --progress --watch --display-error-details --display-reasons --mode development",
     "build": "cross-env NODE_ENV=production webpack --progress --mode production",
     "export": "node bin/export.js",
-    "setup": "npm install && node bin/theme-setup.js"
+    "setup": "npm install && node bin/theme-setup.js",
+    "postinstall": "npm run build"
```

Comment:
Just curious, what would we use this for?

Reply:
Must have been a remnant of the past 😄 I have no idea what this does (I thought that you've done this since the last refactor tbh lol)

Reply:
Nope, doesn't ring a bell. Also I did a quick sanity check and it's not used in the `theme-setup.js` script.